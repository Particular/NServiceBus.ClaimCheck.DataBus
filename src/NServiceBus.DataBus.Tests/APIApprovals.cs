namespace NServiceBus.DataBus.Tests;

using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void ApproveAzureStorageQueueTransport()
    {
        var publicApi = typeof(DataBus).Assembly.GeneratePublicApi(new ApiGeneratorOptions
        {
            ExcludeAttributes = ["System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute"]
        });
        Approver.Verify(publicApi);
    }
}