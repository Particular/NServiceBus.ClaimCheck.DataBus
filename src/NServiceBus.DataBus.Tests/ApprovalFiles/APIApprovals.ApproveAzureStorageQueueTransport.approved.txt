namespace NServiceBus.DataBus
{
    public class DataBus
    {
        public DataBus() { }
        public void TestApi() { }
    }
}